src/components/LocationSelect.tsx

```diff
@@ -0,0 +1,29 @@
+import type { Location } from '../types/sparkeats';
+
+type LocationSelectProps = {
+  locations: Location[];
+};
+
+function LocationSelect({ locations }: LocationSelectProps) {
+  const allCities = locations.map((location) => location.city);
+  const uniqueCities = [...new Set(allCities)];
+
+  return (
+    <div className="location-select">
+      <h2 className="location-select__title">Pick a Location</h2>
+
+      <select className="location-select__menu" name="location" id="location">
+        <option className="location-select__option" value="all">
+          All Places
```

Comment:
See the note about replacing instances of "place" with "location" in the requirements #432!

Reply:
Updated in d69acaf2d50d6cebdcbec0e5865412849eddb683.